test/test_diff.py

```diff
@@ -107,6 +107,84 @@
  delete mode 100644 c/d
 """
 
+TEXT_BLOB1 = """Common header of the file
```

Comment:
I thought it was much simpler to create blobs out of thin air than... _hunt down_ for usable examples in the repos.